src/docs/api/Line.js

```diff
@@ -137,6 +137,17 @@ export default {
         },
       ],
     },
+    {
+      name: 'hide',
+      type: 'Boolean',
+      defaultVal: 'false',
+      isOptional: true,
+      desc: {
+        'en-US': 'Hides the line when true, useful when toggling visibility state via legend',
+        'zh-CN': '将行隐藏为true,在通过图例切换可见性状态时很有用',
```

Comment:
I used an online translator for the `zh-CN` value, so this can probably uses an updated translation from a native speaker.